docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -105,24 +105,52 @@ To test if the Trigger CR is properly connected to the function:
     }
     ```
 
-2. Send an event manually to trigger the function:
+2.  Send an event manually to trigger the function. In the first example, the payload complies with the [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/spec.md) specification and the event is sent directly to Eventing Mesh. If you want to send the events to compatibility layer which forwards them to Eventing Mesh, use the second example. 
+
+   <div tabs name="examples" group="test=trigger">
+    <details>
+    <summary label="CloudEvents">
+    CloudEvents
+    </summary>
 
     ```bash
-    curl -X POST -H "Content-Type: application/json" https://gateway.{CLUSTER_DOMAIN}/$APP_NAME/v1/events -k --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -d \
-    '{
-        "event-type": "{EVENT_TYPE}",
-        "event-type-version": "{EVENT_VERSION}",
-        "event-time": "2020-04-02T21:37:00Z",
-        "data": "123456789"
-    }'
+    curl -v -k --cert {CERT_FILE_NAME} --key {KEY_FILE_NAME} -d '{
+      "specversion": "1.0",
+      "source": "{APP_NAME}",
+      "type": "{EVENT_TYPE}",
+      "eventtypeversion": "{EVENT_VERSION}",
+      "id": "A234-1234-1234",
+      "data": "123456789"
+      }' -H "Content-Type: application/cloudevents+json" https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/events
     ```
+    </details>
+    <details>
+    <summary label="Compatibility layer">
+    Compatibility layer
+    </summary>
+
+    ```bash
+        curl -X POST -H "Content-Type: application/json" https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/events -k --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -d \
+        '{
+            "event-type": "{EVENT_TYPE}",
+            "event-type-version": "{EVENT_VERSION}",
+            "event-time": "2020-04-02T21:37:00Z",
+            "data": "123456789"
+            }'
```

Comment:
A suggestion for you :)

- removed indention
-  the above curl call doesn't have the file extension. So in order to use the same variable let's have a consistent naming :)
- POST is automatically inferred
- Added "-v" to see the status code

```suggestion
curl -H "Content-Type: application/json" https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/events -k --cert {CERT_FILE_NAME} --key {KEY_FILE_NAME} -d \
        '{
            "event-type": "{EVENT_TYPE}",
            "event-type-version": "{EVENT_VERSION}",
            "event-time": "2020-04-02T21:37:00Z",
            "data": "123456789"
         }'
```